src/uct/sm/mm/base/mm_iface.c

```diff
@@ -67,6 +67,30 @@ ucs_config_field_t uct_mm_iface_config_table[] = {
     {"ERROR_HANDLING", "n", "Expose error handling support capability",
      ucs_offsetof(uct_mm_iface_config_t, error_handling), UCS_CONFIG_TYPE_BOOL},
 
+    {"SEND_PRE_OVERHEAD", UCS_VALUE_AUTO_STR,
+     "Time spent after the message request has been passed to the hardware or\n"
+     "system software layers and before operation has been finalized", 0,
+     UCS_CONFIG_TYPE_KEY_VALUE(UCS_CONFIG_TYPE_TIME_UNITS,
+        {"am_short", "send overhead for short Active Message operation type",
```

Comment:
maybe remove "am_" prefix?

Reply:
We check exactly for `am_short`, and `am_bcopy` operation type.
E.g., it will be easier to extend the option by adding overheads for other short/bcopy operatoin types.

Reply:
ok